lib/samson/secrets/vault_server.rb

```diff
@@ -29,8 +29,8 @@ class VaultServer < ActiveRecord::Base
       validate :validate_cert
       validate :validate_connection
 
-      after_save :refresh_vault_clients
-      after_save :expire_secrets_cache
+      after_commit :refresh_vault_clients
+      after_commit :expire_secrets_cache
```

Comment:
What's the difference here?

Reply:
after_save applies to create+update
after_commit applies to create+update+destroy